modules/80_optimization/nash/postsolve.gms

```diff
@@ -323,26 +318,25 @@ if(p80_DevPriceAnticipGlobAllMax("2100") gt 0.1 * p80_surplusMaxTolerance("good"
 );
 
 ***additional criterion: did taxes converge? (only checked if cm_TaxConvCheck is 1)
-p80_taxrev_dev(t,regi) = 0;
-if (cm_TaxConvCheck eq 1,
-  loop(regi,
+p80_convNashTaxrev_iter(iteration,t,regi) = 0;
+loop(regi,
     loop(t,
-      if( abs(vm_taxrev.l(t,regi)) / vm_cesIO.l(t,regi,"inco") gt 1E-4,
-        p80_taxrev_dev(t,regi) = abs(vm_taxrev.l(t,regi)) / vm_cesIO.l(t,regi,"inco");
-        if (t.val lt 2100,
-          s80_bool = 0;
-          p80_messageShow("taxconv") = YES;
-        );
-      );
+         p80_convNashTaxrev_iter(iteration,t,regi) = abs(vm_taxrev.l(t,regi)) / vm_cesIO.l(t,regi,"inco");
+         if (cm_TaxConvCheck eq 1,
+             if( p80_convNashTaxrev_iter(iteration,t,regi) gt 1E-4,
+                 s80_bool = 0;
+                 p80_messageShow("taxconv") = YES;
+             );
+         );
     );
-  );
 );
 
 *** additional criterion: Were regional climate targets reached? 
+p80_emiMktTarget_dev_iter(iteration,ttot,ttot2,ext_regi,emiMktExt) = abs(pm_emiMktTarget_dev(ttot,ttot2,ext_regi,emiMktExt));
 $ifthen.emiMkt not "%cm_emiMktTarget%" == "off" 
 loop((ttot,ttot2,ext_regi,emiMktExt)$pm_emiMktTarget_dev(ttot,ttot2,ext_regi,emiMktExt),
 *** regipol targets must be met within 1% of target deviation, deviation for budget targets is measured relative to target value, while for year targets it is relative to 2015 emissions
-  if((abs(pm_emiMktTarget_dev(ttot,ttot2,ext_regi,emiMktExt)) gt cm_emiMktTarget_tolerance),
+  if((p80_emiMktTarget_dev_iter(iteration,ttot,ttot2,ext_regi,emiMktExt) gt cm_emiMktTarget_tolerance),
```

Comment:
`pm_emiMktTarget_dev_iter` is actually defined and used in module 47. Its name shouldn't change to use `p80` without further refactoring.

Reply:
`pm_emiMktTarget_dev` is defined in module 47, `p80_emiMktTarget_dev_iter` is a new parameter intrduced for the Nash convergance analysis

Reply:
The `p80_emiMktTarget_dev_iter` parameter is duplicating the values of the already existent `pm_emiMktTarget_dev_iter` parameter.
The only difference is that it is applying the absolute transformation to the values, which should be removed in my opinion as it removes information from the parameter that could be used for debugging purposes.

Reply:
ah, wasn't aware of `pm_emiMktTarget_dev_iter`, added it to all realizations to use it here